assets/css/sc-welcome-pg-style.css

```diff
@@ -0,0 +1,69 @@
+#wpwrap {
+	background-color: #f2f7fb;
+}
+
+#simcal-welcome .wp-person a:focus .gravatar,
+#simcal-welcome a:focus,
+#simcal-welcome a:focus .media-icon img {
+	box-shadow: unset;
+}
+
+#simcal-welcome a.button.button-primary {
+	background: rgba(255, 255, 255, 0.2);
+	border-radius: 100px;
+	border: 1px solid white;
+	margin: 6px;
+}
+
+#simcal-welcome .about-text {
+	margin: 15px 0px 0px 0px;
+	color: white;
+}
+
+#simcal-welcome .whats-new-wrap img {
+	border: unset;
+	margin: unset;
+}
+
+#simcal-welcome .about-wrap img {
+	width: unset;
+}
+
+#simcal-welcome .about-wrap .wp-person .gravatar {
+	border-radius: 50%;
+	height: 50px;
+	padding: 0px;
+}
+
+#simcal-welcome .about-wrap .wp-person .web {
+	font-weight: 500;
+	font-size: 14px;
+	color: #000000;
+}
+
+#simcal-welcome .about-wrap .wp-person {
+	width: 23%;
+}
+
+#simcal-welcome a.button.button-primary:hover {
+	background-color: white;
+	color: #60bc4e;
+}
+
+#simcal-welcome .nav-tab-active {
+	color: #3a414c;
+}
+
+#simcal-welcome .wp-person:hover .web {
+	color: #fff;
```

Comment:
Why can't this be done using tailwind class? I think there is `text-white` class, right?

Reply:
i had try but did not work.